spec/lang/step/terminators.md

```diff
@@ -403,6 +472,70 @@ impl<M: Memory> Machine<M> {
 }
 ```
 
+## Catch unwinding
+
+```rust
+impl<M: Memory> Machine<M> {
+    fn eval_terminator(&mut self,
+        Terminator::CatchUnwind{try_fn, data_ptr, catch_fn, ret: ret_expr, next_block}: Terminator
+    ) -> NdResult {
+        // Evaluate the return place.
+        let catch_unwind_ret = self.eval_place(ret_expr)?;
+
+        // Evaluate the function pointers of `try_fn` and `catch_fn`.
+        let try_fn = self.eval_function_ptr(try_fn)?;
+        let catch_fn = self.eval_function_ptr(catch_fn)?;
+
+        // Prepare the argument for the call.
+        let data_ptr = self.eval_argument(ArgumentExpr::ByValue(data_ptr))?;
+        let args = list!(data_ptr);
+
+        // Define the catch action.
+        let catch_action = CatchAction{
```

Comment:
```suggestion
        let catch_action = CatchAction {
```
Please do a thorough pass over your PR regarding these style issues, there are more of them and I'm not going to manually mark them all.